src/shared/utils.tsx

```diff
@@ -0,0 +1,22 @@
+import moment from "moment";
+
+class Utils {
+
+    static formatDate(dateStr: string): string {
+        const date = new Date(dateStr);
+
+        return moment(date).format("YYYY MM DD");
+    }
+
+    static formatGender(gender: string): string {
+        return gender.includes('F') ? 'Female' : 'Male';
+    }
+
+    static calculatePatientAge(patientDOB: string): number {
+        const today = new Date();
+        const dob = new Date(patientDOB);
+        return today.getFullYear() - dob.getFullYear();
+    }
+}
```

Comment:
For consistency purposes, please, stick to 2 space indentations.

Reply:
DIdn't realize tab spacing defaulted back to 2. Fixed!